test/e2e/framework.go

```diff
@@ -38,13 +38,31 @@ type Framework struct {
 
 	Namespace *api.Namespace
 	Client    *client.Client
+
+	nsCreateFunc func(string, *client.Client) (*api.Namespace, error)
 }
 
 // NewFramework makes a new framework and sets up a BeforeEach/AfterEach for
 // you (you can write additional before/after each functions).
 func NewFramework(baseName string) *Framework {
```

Comment:
probably better to have this call the new function, passing `createTestingNS`, rather than duplicating the code.